modules/osdk-prepare-supported-images.adoc

```diff
@@ -1,30 +1,33 @@
 // Module included in the following assemblies:
 //
 // * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+:osdk_ver: v1.3.0
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+:type: Helm
+endif::[]
 
 [id="osdk-prepare-supported-images_{context}"]
-= Preparing your Operator use supported images
+= Preparing your Operator to use supported images
 
-Before running your Go-based Operator on {product-title}, update your project to use supported images.
+Before running your {type}-based Operator on {product-title}, update your project to use supported images.
 
 .Procedure
 
+ifdef::golang[]
 . Update the following lines in the project root-level Dockerfile to use supported images.
 
-.. Change the default builder image reference from:
-+
-[source,terminal]
-----
-golang:1.15
-----
-+
-to:
-+
-[source,terminal]
-----
-registry.redhat.io/openshift4/builder:rhel-8-golang-1.15-openshift-4.7
```

Comment:
Removing per discussion with @jmrodri and @estroz, as there is currently no downstream Go 1.15 builder to use.